website/addons/wiki/views.py

```diff
@@ -140,16 +140,15 @@ def wiki_widget(**kwargs):
     wiki = node.get_addon('wiki')
     wiki_page = node.get_wiki_page('home')
 
-    more = False
+    more = True if len(node.wiki_pages_current.keys()) >= 2 else False
```

Comment:
Be wary of ternary expressions that look like

```python
my_cool_variable = True if some_boolean else False
```

because it can be more simply expressed as

```python
my_cool_variable = some_boolean
```